governance/votes/Executive vote - March 4, 2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] Core Unit Budget Distributions, Recognized Delegate Compensation, MOMC Parameter Changes - March 4, 2022
+summary: Core Unit Budget Distributions, Recognized Delegate Compensation Distribution for February, MOMC Parameter Changes for WSTETH-A and DIRECT-AAVEV2-DAI
```

Comment:
Complete with period? Little weirder for these longer summaries, but that has been our standard